### PR TITLE
updated the testnet description doc [sc-6827]

### PR DESCRIPTION
**Pull request by kbelita**
[sc-6827](https://app.shortcut.com/rotational/story/6827/trisa-dev-testnet-updates)

* updated description
* changed shortcode link pointing to TestNet architecture image to markdown image.

### CODE REVIEW

File: docs/content/testnet/_index.en.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+
+![TestNet Architecture](/img/testnet_architecture.png)
+
+## Joining the TestNet
+
+The following steps are required to join the TestNet:
+
+1. [Register](https://vaspdirectory.net/certificate/registration) with the GDS to create your TRISA Account with your VASP email address, where you can opt-in for TestNet Certificates. During registration, you can add collaborators within your organization.
+
+2. Complete the  multi-part VASP Verification form and due diligence process. Once approved, you gain will gain to the TestNet.
```
Comment (eschmier) [inline docs/content/testnet/_index.en.md:30]: Typo: "Once approved, you will gain access to the TestNet."

File: docs/content/testnet/_index.en.md [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+
+3. Set up your TRISA node or integrate with a 3rd-party solution to start testing. When considering setting up your own server to host your own TRISA node, you must consider items necessary that may incur significant costs and resources, such as the server itself, long-term data storage solution, and developer time to configure and test.
```
Comment (eschmier) [inline docs/content/testnet/_index.en.md:32]: Reworded slightly plus fixed typo: "Set up your TRISA node or integrate with a 3rd-party solution to start testing. When considering setting up your own server to host your own TRISA node, you must consider resources necessary that may incur costs such as the server itself, a long-term data storage solution, and developer time to configure and test.

File: docs/content/testnet/_index.en.md [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
 - [TestNet Demo](https://vaspbot.net) - a demo site to show TRISA interactions between “robot” VASPs that run in the TestNet
 
-The TestNet also hosts three robot VASPs or rVASPs that have been implemented as a convenience for TRISA members to integrate their TRISA services. The primary rVASP is Alice, a secondary for demo purposes is Bob, and to test interactions with non-verified TRISA members, there is also an "evil" rVASP.
\ No newline at end of file
+The TestNet hosts three ["robot VASPs" (rVASPs)]({{< relref "testnet/rvasps">}}) that have been implemented as a convenience for TRISA members to integrate their TRISA services and validate the compliance solution safely. The primary rVASP is Alice, a secondary for demo purposes is Bob, and an "evil" rVASP to test interactions with non-verified TRISA members.
+
+The TestNet also provides a [command line utility]({{< relref "testnet/trisa-cli">}}) for interacting with the API for administrative and debugging purposes, using the testnet certificates.
+
+![TestNet Architecture](/img/testnet_architecture.png)
```
Comment (eschmier) [inline docs/content/testnet/_index.en.md:22]: I wasn't able to see the image file so can't tell if it works.

**Review by eschmier — CHANGES_REQUESTED** [review]
Looks good @kbelita ! Thank you for giving attention to this page. I fixed a few small typos and revised a sentence.

**Review by rebeccabilbro — APPROVED** [review]
Looks great @kbelita and thank you @eschmier for the helpful feedback!

**Pull request merged**